pkg/universe.dagger.io/alpha/go/goreleaser/test/image.cue

```diff
@@ -0,0 +1,44 @@
+package goreleaser
+
+import (
+	"dagger.io/dagger"
+
+	"universe.dagger.io/docker"
+	"universe.dagger.io/alpha/go/goreleaser"
```

Comment:
@jpadams I tried leaving the import, but I couldn't run the tests locally that way.

Reply:
WDYT @helderco?

Reply:
I encourage to import the package because it's how users will do, but that should not be mandatory.

Reply:
The import looks good to me.

I cannot seem to find the `test.bats` file that actually runs these tests. Here is the go bats file: https://github.com/dagger/dagger/blob/v0.2.23/pkg/universe.dagger.io/go/test/test.bats . This is how that actually runs in CI: https://github.com/dagger/dagger/blob/2927c77876a83c0aa688ba648fc376f31eccf5bf/ci.cue#L148-L154